tests/testAutopas/RegionParticleIteratorTest.cpp

```diff
@@ -65,22 +103,66 @@ TEST_F(RegionParticleIteratorTest, testLinkedCellsRegionParticleIteratorBehavior
   for (auto iterator = lcContainer.getRegionIterator(testRegionMin, _regionMax, autopas::IteratorBehavior::haloOnly);
        iterator.isValid(); ++iterator) {
     iterator->touch();
+    EXPECT_TRUE(utils::inBox(iterator->getR(), testRegionMin, _regionMax)
```

Comment:
Create a bool for this condition for readability (especially of gtest output on case of fail)

Reply:
https://github.com/AutoPas/AutoPas/pull/143/commits/9c1a93fa659df430846823bb885d3291ad059715